db/seeds.rb

```diff
@@ -13,13 +13,17 @@
 Instance.find_or_initialize_by(name: 'Default', host: '*').save!(validate: false)
 
 ActsAsTenant.with_tenant(Instance.default) do
-  # Create the Coursemology system account.
-  user = User.exists?(User::SYSTEM_USER_ID)
-  unless user
+  # Create the Coursemology default user accounts.
+  unless User.exists?(User::SYSTEM_USER_ID)
     User.create!(id: User::SYSTEM_USER_ID, name: 'System')
   end
+  unless User.exists?(User::DELETED_USER_ID)
+    User.new(id: User::DELETED_USER_ID, name: 'Deleted').save!
+  end
+  # Sequence were outdated when we specify the id.
+  ActiveRecord::Base.connection.reset_pk_sequence!(User.table_name)
```

Comment:
Okay, using negative PK now. Btw, is it a convention to use negative id for preserved records ?